webapp/app/components/pages/home/HomePage.jsx

```diff
@@ -46,13 +45,25 @@ export class HomePage extends React.Component {
     const selectRow = {
       mode: 'radio',
       clickToSelect: true,
-      bgColor: '#c8e6c9'
+      bgColor: '#c8e6c9',
+      selected: this.state.selectedUser,
+      onSelect: this.setSelectedRow
     };
     
     return (
       <div>
         <Header />
         <div className="container">
+          <Row>
+            <Col md="8">
+              <h4>Users List</h4>
+            </Col>
+            <Col md="4">
+              <Button color="primary">Add</Button>{' '}
```

Comment:
rather than adding computed blank spaces after each button `{' '}` I would add the desired spacing using css 
`.user-actions > button {
    margin-right: 8px;
}

.user-actions > button::last-child {
    margin-right: 0;
}
`

Reply:
Good catch, this is copy pasted code ;-)